src/Microsoft.ML.CpuMath/CpuMathUtils.netcoreapp.cs

```diff
@@ -24,6 +24,14 @@ public static partial class CpuMathUtils
         public static int GetVectorAlignment()
             => Avx.IsSupported ? Vector256Alignment : (Sse.IsSupported ? Vector128Alignment : FloatAlignment);
 
+        /// <summary>
+        /// Multiplies a matrix times a source.
+        /// </summary>
+        /// <param name="transpose"></param>
```

Comment:
If you leave the params here, you'd need to put some explanation text on them.

Reply:
@sfilipi The params are now documented. Though, I'm sure some of them don't offer much value.